apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -515,7 +517,7 @@ const BookingPage = ({
                       <FormattedNumber
                         value={eventType.price / 100.0}
                         style="currency"
-                        currency={eventType.currency.toUpperCase()}
+                        currency={currency?.toUpperCase() || eventType.currency.toUpperCase()}
```

Comment:
Instead of passing the currency manually, let's focus on figure out why the eventType doesn't have the correct currency and:

1. See if it can be resolved at runtime, or..
2. If the DB is wrong see if we can resolve it with a single migration